.github/build.yaml.gomplate

```diff
@@ -67,26 +67,37 @@
   {{{ $config := (datasource "config") }}}
   {{{- if $config.local_runner }}}
     runs-on: [{{{ join $config.runs_on ", "}}}]
-    if: contains(["mudler", "davidcassany", "itxaka", kkaempf", "cOS-cibot"], "${{ github.actor }}") || contains(github.event.pull_request.labels.*.name, 'safe to test')
   {{{- else }}}
     runs-on: ubuntu-latest
   {{{ end }}}
 {{{end}}}
 
-{{{define "needs"}}}
-    {{{ $config := (datasource "config") }}}
-    {{{if has $config "labels"}}}
-    needs:
-      - check-labels
-    {{{end}}}
+{{{- define "generate_labels"}}}
+{{{- $config := (datasource "config") }}}
+{{{- range $config.labels}}} && contains(github.event.pull_request.labels.*.name, '{{{ . }}}'){{{- end}}}
+{{{- end}}}
+
+{{{define "if_with_labels"}}}
+{{{ $config := (datasource "config") }}}
+{{{if has $config "labels"}}}
+  {{{- if $config.local_runner }}}
+    if: contains(fromJson('["mudler", "davidcassany", "itxaka", "kkaempf", "cOS-cibot"]'), github.actor) {{{- tmpl.Exec "generate_labels" }}}
```

Comment:
That's neat, guessing that later on we could switch to handle users in the same way for labels

Reply:
would be great if we could automagically get the members of the cos team and use it here :D